lib/utils/config-reader.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const debug = require('debug')('exam:config-reader');
+
+const potentialConfigFiles = ['testem.js', 'testem.json'];
+
+/**
+ * Given an array of file paths, returns the first one that exists and is
+ * accessible. Paths are relative to the process' cwd.
+ *
+ * @param {Array<string>} files
+ * @return {string} file
+ */
+function _findValidFile(files) {
+  for (let i = 0; i < files.length; i++) {
+    // TODO: investigate this cwd() usually they are in-error...
```

Comment:
What does this mean?

Reply:
@choheekim do you remember what this TODO refers to again?

Reply:
if memory serves, the `cwd`  was in the existing code. I pointed out, that it is not usually appropriate to use `cwd` in a library, rather it should be parameterized. 

But given this was existing code, merely adapted for this new case, its not the end of the world. Although I would like to encourage someone clean this up later.

Reply:
I think the `TODO` has been there before this pr. I will be investigating to clean this up